azure-pipelines.yml

```diff
@@ -94,5 +94,11 @@ jobs:
     - script: pip install -r source/requirements.txt
       displayName: 'Install Python Requirements'
 
+    - script: sudo apt-get update
+      displayName: 'Update System Packages'
+
+    - script: sudo apt-get install -y librsvg2-bin
+      displayName: 'Install SVG requirements'
+
```

Comment:
Do we need this block? it should've been installed above.

Reply:
It only installs for the build script, linkcheck for some reason runs librsvg2